environment.yml

```diff
@@ -14,17 +14,18 @@ dependencies:
   - pytorch=1.12.1
   - torchvision=0.13.1
   - cudatoolkit=11.6
-  - pytorch-lightning=1.7.7
-  - torchmetrics=0.9.2
   - parameterized=0.8.1
   - pip:
+      - pytorch-lightning==1.8.4.post0
```

Comment:
Why is the version with postfix?

Reply:
[Check here](https://github.com/Lightning-AI/lightning/releases)